lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -108,6 +108,7 @@
     return Date.now();
   }
   helloDate();
+  const d = Date.now(); // FAIL
   eval('Date.now()'); // FAIL
```

Comment:
Not much,  just wanted to check that `Date.now()` outside of an inner function was also producing results. Until smokehouse audits number of violations, this won't do much other than a manual verify.